contracts/Quest.sol

```diff
@@ -53,26 +53,31 @@ contract Quest is OwnableUpgradeable, IQuest {
     function start() public virtual onlyOwner {
         isPaused = false;
         hasStarted = true;
+        emit Started(block.timestamp);
     }
 
     /// @notice Pauses the Quest
     /// @dev Only the owner of the Quest can call this function. Also requires that the Quest has started (not by date, but by calling the start function)
     function pause() external onlyOwner onlyStarted {
         isPaused = true;
+        emit Paused(true);
     }
 
     /// @notice Unpauses the Quest
     /// @dev Only the owner of the Quest can call this function. Also requires that the Quest has started (not by date, but by calling the start function)
     function unPause() external onlyOwner onlyStarted {
         isPaused = false;
+        emit Paused(false);
     }
 
     /// @notice Marks token ids as claimed
     /// @param tokenIds_ The token ids to mark as claimed
     function _setClaimed(uint256[] memory tokenIds_) private {
-        for (uint i = 0; i < tokenIds_.length;) {
+        for (uint i = 0; i < tokenIds_.length; ) {
```

Comment:
Unnecessary trailing whitespace here after the semicolon?

Reply:
It's intentional. I have this plugin formatting the solidity files: https://marketplace.visualstudio.com/items?itemName=NomicFoundation.hardhat-solidity